extensions/Worker.Extensions.Http/src/HttpResultAttribute.cs

```diff
@@ -2,14 +2,15 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
 
 namespace Microsoft.Azure.Functions.Worker
 {
     /// <summary>
     /// Attribute used to mark an HTTP Response on an HTTP Trigger function with multiple output bindings.
     /// </summary>
     [AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
-    public sealed class HttpResultAttribute : Attribute
+    public sealed class HttpResultAttribute : OutputBindingAttribute
```

Comment:
More restrictive usage than output binding (see line above). Having this implement `OutputBindingAttribute` means we can prevent multiple "output" stacking (i.e. making a property both http result and queue output) using the current code.